### PR TITLE
New package: libpurple-hangouts-20180704.

### PR DESCRIPTION
**Pull request by pbui**

### CODE REVIEW

File: srcpkgs/purple-hangouts/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+pkgname=purple-hangouts
+version=20180704
+revision=1
+wrksrc="EionRobb-$pkgname-$_hghash"
+build_style=gnu-makefile
+hostmakedepends="pkg-config unzip"
+makedepends="libpurple-devel protobuf-c-devel json-glib-devel"
+short_desc="Google Hangouts plugin for libpurple [hg $_hghash]"
+maintainer="Peter Bui <pbui@github.bx612.space>"
+license="GPL-3"
```
Comment (abenson) [inline srcpkgs/purple-hangouts/template]: GPL-3.0-or-later

**Comment by abenson** [comment]
The other plugins are `libpurple-[pluginname]`, should this one follow that trend?

**Comment by pbui** [comment]
@abenson Yes, it should follow that trend. I'll make those changes.

**Comment by pbui** [comment]
@abenson I've updated the license, changed the package name, and fixed cross building (I think).

**Pull request merged**